src/frontend/src/flows/register/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Connection } from "../../utils/iiConnection";
+import { unknownToString } from "../../utils/utils";
+import { confirmRegister } from "./captcha";
+import { makeCaptcha } from "./captcha";
+import { LoginFlowResult, cancel } from "../login/flowResult";
+import { constructIdentity } from "./construct";
+import { promptDeviceAlias } from "./alias";
+
+/** Registration (anchor creation) flow for new users */
+export const register = async ({
+  connection,
+}: {
+  connection: Connection;
+}): Promise<LoginFlowResult> => {
+  try {
+    const alias = await promptDeviceAlias();
+    if (alias === null) {
+      return cancel;
+    }
+
+    const [captcha, identity] = await Promise.all([
+      makeCaptcha(connection),
+      constructIdentity(),
+    ]);
+
+    const result = await confirmRegister(
```

Comment:
(nit, because was this way previously) The captcha module still contains the part about congratulating the user. I would have expected it here, as it has nothing to do with captcha.